fe/java-udf/src/main/java/org/apache/doris/udf/JdbcExecutor.java

```diff
@@ -325,277 +325,233 @@ private void init(String driverUrl, String sql, int batchSize, String driverClas
     public void copyBatchBooleanResult(Object columnObj, boolean isNullable, int numRows, long nullMapAddr,
             long columnAddr) {
         Boolean[] column = (Boolean[]) columnObj;
-        byte[] columnData = new byte[numRows];
         if (isNullable) {
-            byte[] nullMap = new byte[numRows];
             for (int i = 0; i < numRows; i++) {
                 if (column[i] == null) {
-                    nullMap[i] = 1;
+                    UdfUtils.UNSAFE.putByte(nullMapAddr + i, (byte) 1);
```

Comment:
should always putByte. if column[i] != null, put 0

Reply:
add memset(null_map.data(), 0, num_rows) in be